templates/zerver/help/mention-a-team-member.md

```diff
@@ -0,0 +1,11 @@
+#Mention a Team Member
+
+![Mention a Team Member](/static/images/help/mention_user.png)
+
+When having a conversation in Zulip, it is sometimes necessary to call the attention of another member. You can alert ("``@mention``") other users with a particular stream-message from within the body of the message itself. Type @ and their email address or one of their names, and the system will offer you tab-completions. The completed ``@mention`` will look something like: ``@**name**``, and it will appear as ``@name`` in your posted message.
```

Comment:
I think "the beginning of their email address" would be a bit clearer, since now it seems like you have to type the whole address which is not the case.

I'm also not sure you have to use "``", since in the `index.md` only `**@**` has been used, for instance instead of ``@**name**`` you'd have **@name**.